docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-rdp-enabled-via-registry.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[prebuilt-rule-0-16-2-rdp-enabled-via-registry]]
+=== RDP Enabled via Registry
+
+Identifies registry write modifications to enable Remote Desktop Protocol (RDP) access. This could be indicative of adversary lateral movement preparation.
```

Comment:
```suggestion
Identifies registry write modifications to enable Remote Desktop Protocol (RDP) access. This could be indicative of an adversary preparing for lateral movement.
```